app/src/test/java/org/mozilla/fenix/settings/logins/SyncPreferenceViewTest.kt

```diff
@@ -104,55 +126,57 @@ class LoginsSyncPreferenceViewTest {
     }
 
     @Test
-    fun `show status for existing account`() {
+    fun `GIVEN LoginScreen and syncLogins true WHEN updateSyncPreferenceStatus THEN setStatus false`() {
+        // Given
         every { accountManager.authenticatedAccount() } returns mockk()
         every { accountManager.accountNeedsReauth() } returns false
-        createView()
+        every { anyConstructed<SyncEnginesStorage>().getStatus() } returns mapOf(
+            SyncEngine.Passwords to true
+        )
+        every { anyConstructed<SyncEnginesStorage>().setStatus(any(), any()) } just Runs
 
-        verify { syncLoginsPreference.summary = "Off" }
-        assertTrue(clickListener.captured.onPreferenceClick(syncLoginsPreference))
+        // When
+        createView()
 
-        verify {
-            navController.navigate(
-                SavedLoginsAuthFragmentDirections.actionGlobalAccountSettingsFragment()
-            )
-        }
+        // Then
+        verify { syncPreference.isSwitchWidgetVisible = true }
+        verify { syncPreference.isChecked = true }
+        assertTrue(preferenceChangeListener.captured.onPreferenceChange(syncPreference, false))
+        verify { anyConstructed<SyncEnginesStorage>().setStatus(any(), false) }
     }
 
     @Test
-    fun `show status for existing account with passwords`() {
-        every { anyConstructed<SyncEnginesStorage>().getStatus() } returns mapOf(
-            SyncEngine.Passwords to true
-        )
+    fun `GIVEN LoginScreen and syncLogins false WHEN updateSyncPreferenceStatus THEN setStatus true`() {
+        // Given
```

Comment:
I think we can do without the Given, When, Then inline comments since the code is quite self explanatory here due to the new line usage to group the logic very nicely to the Given, When, Then sections that can be implicitly implied by reading it. I would not mind if you also grouped the other tests similarly.

Reply:
> I think we can do without the Given, When, Then inline comments

👍 

> I would not mind if you also grouped the other tests similarly.

Modifying other test methods will override the commit annotations, making tracking down the original commits more difficult, as a rule I try to abstain from doing that.

Reply:
That's fine, but my ask here was just to add new lines to create that Given, When, Then separation for the every {}, and verify() which you have do for these tests. I would not expect adding new lines to hurt too much with commit history since the original logic will remain intact.

Reply:
Done 😄